.github/workflows/renovate-validate.yaml

```diff
@@ -0,0 +1,12 @@
+name: Validate renovate config
+on:
+  pull_request: {}
```

Comment:
we should filter this by path, so that it doesn't run on every PR

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

Reply:
TIL!